lib/passageidentity/client.rb

```diff
@@ -92,30 +93,19 @@ def initialize(app_id:, api_key: "", auth_strategy: COOKIE_STRATEGY)
     end
 
     def get_connection
-      if @api_key == ""
-        @connection =
-          Faraday.new(url: @api_url) do |f|
-            f.request :json
-            f.request :retry
-            f.response :raise_error
-            f.response :json
-            f.adapter :net_http
-          end
-      else
-        @connection =
-          Faraday.new(
-            url: @api_url,
-            headers: {
-              "Authorization" => "Bearer #{@api_key}"
-            }
-          ) do |f|
-            f.request :json
-            f.request :retry
-            f.response :raise_error
-            f.response :json
-            f.adapter :net_http
-          end
-      end
+      gemspec = File.join(__dir__, "../../passageidentity.gemspec")
+      spec = Gem::Specification.load(gemspec)
+      headers = { "Passage-Version" => "ruby #{spec.version}" }
```

Comment:
☝️ uses "ruby" as the identifier, because the `#{spec.name}` value is just "passageidentity", which can be ambiguous.

Reply:
can we add the version to this as well?

Reply:
This will pass the version through; see the logs [here](https://cloudlogging.app.goo.gl/MSxXFiFJThnT6VvA9) for an example.

Does it make more sense to make this
```suggestion
      headers = { "Passage-Version" => "passage-ruby #{spec.version}" }
```
to match the repo name?

Reply:
ignore me I somehow only saw `#{spec.name}`